crux-test/project.clj

```diff
@@ -33,6 +33,7 @@
                  [org.clojure/test.check "0.10.0"]
                  [org.slf4j/slf4j-api "1.7.26"]
                  [ch.qos.logback/logback-classic "1.2.3"]
+                 [org.codehaus.janino/janino "2.6.1"]
```

Comment:
what's this?

Reply:
Left over from trying to get it in memory, good catch